jobs/JFV3_CYCLONE_GENESIS_JP

```diff
@@ -51,54 +45,48 @@ export SCRIPTens_tracker=${SCRIPTens_tracker:-$HOMEens_tracker/scripts}
 ##############################
 # Run setpdy and initialize PDY variables
 ##############################
-#setpdy.sh
-#. PDY
-export PDY=20230413
-
+setpdy.sh
+. PDY
 ##############################################
 # Define COM directories
 ##############################################
 
 if [[ -d /scratch2 ]] ; then
   # We are on NOAA Hera
-#  export COMINgfs=${COMINgfs:-${COMROOTp1}/gfs/prod/gfs.${PDY}}
-#  export COMINsyn=${COMINsyn:-${COMROOTp1}/arch/prod/syndat}
-#  export COMINsyn=/scratch1/NCEPDEV/hwrf/noscrub/input/SYNDAT
-  export COMINsyn=/scratch2/NCEPDEV/ensemble/Jiayi.Peng/noscrub/MetTCData/TCvital
-  export COMINgfs=/scratch2/NCEPDEV/ensemble/Jiayi.Peng/noscrub/ptmp/gfs.${PDY}
+  export COMINsyn=${COMINsyn:-/scratch2/NCEPDEV/ovp/Hana.Jafary/Input/TCvital}
+  export COMINgfs=${COMINgfs:-/scratch2/NCEPDEV/ovp/Hana.Jafary/Input/gfs.${PDY}}
 
-elif [[ -d /work ]] ; then
+elif [[ -d /work2/noaa ]] ; then
   # We are on MSU Orion
```

Comment:
This could be either Orion or Hercules.  This is a technicality in this case, but in some cases it does matter.  If you need to know when one you are on, you can use this trick:

```
  # We are on MSU Orion or Hercules
  mount=$(findmnt -n -o SOURCE /home)
  if [[ ${mount} =~ "hercules" ]]; then
    MACHINE_ID=hercules
  else
    MACHINE_ID=orion
  fi
```

Also, it may be useful at some point (not for this PR) to develop a `detect_machine.sh` script that is sourced everywhere that needs it.  This will save having to edit these kinds of things across multiple scripts.  See, for example, https://github.com/NOAA-EMC/gfs-utils/blob/develop/ush/detect_machine.sh.